shared/common-adapters/chat-button.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+import * as ChatConstants from '../constants/chat2'
+import * as Chat2Gen from '../actions/chat2-gen'
+import * as ConfigGen from '../actions/config-gen'
+import * as Container from '../util/container'
+import * as Styles from '../styles'
+import Icon from './icon'
+import WaitingButton from './waiting-button'
+
+const Kb = {
```

Comment:
lets just put this in chat and not in common-adapters. i think we need to reduce whats actually in here